### DISCUSSION TITLE
Default shape size

### DISCUSSION THREAD

**Discussion by grationd** [jgraph/drawio#3362] [Q&A]
Hello everyone,

I'm new to draw.io and I'm wondering if there is anyway to set the size of a certain shape as default. If that's not a feature then it'd be greatly appreciated to implement this small but useful change in future releases. 

Thank you in advance

**Comment by davidjgraph** [comment]
Yes, you can set the size of a certain shape as default. To do this, select the shape that you want to set the default size for, then adjust the size of the shape using the handles or by specifying the dimensions in the format panel on the right-hand side of the editor. Once you have the size you want, right-click on the shape and select 'Set as Default Style' from the context menu.

If you have multiple shapes that you want to set the default size for, you can also group them together and set the default style for the group.

**Reply by grationd** [reply]
I tried this method and it didn't work. It still shows the default size when I drag and drop the shape afterwards. The only thing it seems to be doing is that it saves changed colors and applies it to all the shapes on the left panel. I'm using the desktop version but I also tried the web version.

**Comment by kxm187** [comment]
I know this is an old thread, but I had the same issue, and I figured it.  Put the item on your drawing and scale it to the size you want.  Then right click and choose "Add to Scratchpad".  It will be added to your Scratchpad Shapes menu.  From now on, use the version in your Scratchpad and the shapes will default to the size they were when you added it.
